### ISSUE TITLE
Don't install on Ubuntu 16.04 with Python 3

### ISSUE THREAD

**Issue by ArsenBespalov** [yandex/yandex-tank#524]
`$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.4 LTS
Release:	16.04
Codename:	xenial`

After try install from repository:

`$ sudo -H pip install https://api.github.com/repos/yandex/yandex-tank/tarball/master
Collecting https://api.github.com/repos/yandex/yandex-tank/tarball/master
  Downloading https://api.github.com/repos/yandex/yandex-tank/tarball/master (1.9MB)
    100% |████████████████████████████████| 1.9MB 786kB/s
Collecting psutil>=1.2.1 (from yandextank==1.9.9)
  Using cached psutil-5.4.3.tar.gz
Requirement already satisfied: requests>=2.5.1 in /usr/lib/python3/dist-packages (from yandextank==1.9.9)
Collecting paramiko>=1.16.0 (from yandextank==1.9.9)
  Using cached paramiko-2.4.0-py2.py3-none-any.whl
Collecting pandas>=0.18.0 (from yandextank==1.9.9)
  Using cached pandas-0.22.0-cp35-cp35m-manylinux1_x86_64.whl
Collecting numpy>=1.12.1 (from yandextank==1.9.9)
  Using cached numpy-1.14.1-cp35-cp35m-manylinux1_x86_64.whl
Collecting future>=0.16.0 (from yandextank==1.9.9)
  Using cached future-0.16.0.tar.gz
Requirement already satisfied: pip>=8.1.2 in /usr/local/lib/python3.5/dist-packages (from yandextank==1.9.9)
Collecting matplotlib>=1.5.3 (from yandextank==1.9.9)
  Using cached matplotlib-2.1.2-cp35-cp35m-manylinux1_x86_64.whl
Collecting seaborn>=0.7.1 (from yandextank==1.9.9)
  Using cached seaborn-0.8.1.tar.gz
Requirement already satisfied: pyyaml>=3.12 in /usr/local/lib/python3.5/dist-packages (from yandextank==1.9.9)
Collecting cerberus>=1.1 (from yandextank==1.9.9)
  Using cached Cerberus-1.1.tar.gz
Collecting influxdb>=5.0.0 (from yandextank==1.9.9)
  Using cached influxdb-5.0.0-py2.py3-none-any.whl
Collecting pynacl>=1.0.1 (from paramiko>=1.16.0->yandextank==1.9.9)
  Using cached PyNaCl-1.2.1-cp35-cp35m-manylinux1_x86_64.whl
Requirement already satisfied: pyasn1>=0.1.7 in /usr/local/lib/python3.5/dist-packages (from paramiko>=1.16.0->yandextank==1.9.9)
Collecting bcrypt>=3.1.3 (from paramiko>=1.16.0->yandextank==1.9.9)
  Using cached bcrypt-3.1.4-cp35-cp35m-manylinux1_x86_64.whl
Requirement already satisfied: cryptography>=1.5 in /usr/lib/python3/dist-packages (from paramiko>=1.16.0->yandextank==1.9.9)
Requirement already satisfied: python-dateutil>=2 in /usr/local/lib/python3.5/dist-packages (from pandas>=0.18.0->yandextank==1.9.9)
Requirement already satisfied: pytz>=2011k in /usr/lib/python3/dist-packages (from pandas>=0.18.0->yandextank==1.9.9)
Requirement already satisfied: six>=1.10 in /usr/local/lib/python3.5/dist-packages (from matplotlib>=1.5.3->yandextank==1.9.9)
Collecting cycler>=0.10 (from matplotlib>=1.5.3->yandextank==1.9.9)
  Using cached cycler-0.10.0-py2.py3-none-any.whl
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in /usr/lib/python3/dist-packages (from matplotlib>=1.5.3->yandextank==1.9.9)
Collecting scipy (from seaborn>=0.7.1->yandextank==1.9.9)
  Downloading scipy-1.0.0-cp35-cp35m-manylinux1_x86_64.whl (49.6MB)
    99% |████████████████████████████████| 49.6MB 45.6MB/s eta 0:00:01Killed`

Where is error? How to install?

**Comment by direvius** [comment]
I don't see any errors in log you've provided. What's the problem?

**Comment by ArsenBespalov** [comment]
Yep! I too don't see any errors, just break download process on 99% at scipy-1.0.0-cp35-cp35m-manylinux1_x86_64.whl package.

**Comment by nnugumanov** [comment]
scipy is a third-party library, so try something from this
https://www.google.ru/search?newwindow=1&client=safari&rls=en&dcr=0&ei=u7GfWs_bDsbGkwWJubDIAw&q=scipy+installation+has+been+killed&oq=scipy+installation+has+been+killed&gs_l=psy-ab.3...1241.1241.0.1613.1.1.0.0.0.0.119.119.0j1.1.0....0...1.1.64.psy-ab..0.0.0....0.RcDvq-Fu85o
